administrate/metrics/overview.md

```diff
@@ -241,229 +220,319 @@ Telegraf provide lots of metrics described in their [documentation](https://gith
+      <td align="center">kernel.context_switches</td>
+      <td align="center">netstat.tcp_closing</td>
+    </tr>
+    <tr>
+      <td align="center">kernel.entropy_avail</td>
+      <td align="center">netstat.tcp_established</td>
+    </tr>
+    <tr>
+      <td align="center">kernel.interrupts</td>
+      <td align="center">netstat.tcp_fin_wait1</td>
+    </tr>
+    <tr>
+      <td align="center">kernel.processes_forked</td>
+      <td align="center">netstat.tcp_fin_wait2</td>
+    </tr>
+    <tr>
+      <td align="center">mem.active</td>
+      <td align="center">netstat.tcp_last_ack</td>
+    </tr>
+    <tr>
+      <td align="center">mem.available</td>
+      <td align="center">netstat.tcp_listen</td>
+    </tr>
+    <tr>
+      <td align="center">mem.available_percent</td>
+      <td align="center">netstat.tcp_none</td>
+    </tr>
+    <tr>
+      <td align="center">mem.buffered</td>
+      <td align="center">netstat.tcp_syn_recv</td>
+    </tr>
+    <tr>
+      <td align="center">mem.cached</td>
+      <td align="center">netstat.tcp_syn_sent</td>
+    </tr>
+    <tr>
+      <td align="center">mem.commit_limit</td>
+      <td align="center">netstat.tcp_time_wait</td>
+    </tr>
+    <tr>
+      <td align="center">mem.committed_as</td>
+      <td align="center">netstat.udp_socket</td>
+    </tr>
+    <tr>
+      <td align="center">mem.dirty</td>
+      <td align="center">processes.blocked</td>
+    </tr>
+    <tr>
+      <td align="center">mem.free</td>
+      <td align="center">processes.dead</td>
+    </tr>
+    <tr>
+      <td align="center">mem.high_free</td>
+      <td align="center">processes.idle</td>
+    </tr>
+    <tr>
+      <td align="center">mem.high_total</td>
+      <td align="center">processes.paging</td>
+    </tr>
+    <tr>
+      <td align="center">mem.huge_page_size</td>
+      <td align="center">processes.running</td>
+    </tr>
+    <tr>
+      <td align="center">mem.huge_pages_free</td>
+      <td align="center">processes.sleeping</td>
+    </tr>
+    <tr>
+      <td align="center">mem.huge_pages_total</td>
+      <td align="center">processes.stopped</td>
+    </tr>
+    <tr>
+      <td align="center">mem.inactive</td>
+      <td align="center">processes.total</td>
+    </tr>
+    <tr>
+      <td align="center">mem.low_free</td>
+      <td align="center">processes.total_threads</td>
+    </tr>
+    <tr>
+      <td align="center">mem.low_total</td>
+      <td align="center">processes.unknown</td>
+    </tr>
+    <tr>
+      <td align="center">mem.mapped</td>
+      <td align="center">processes.zombies</td>
+    </tr>
+    <tr>
+      <td align="center">mem.page_tables</td>
+      <td align="center">procstat_lookup.pid_count</td>
+    </tr>
+    <tr>
+      <td align="center">mem.shared</td>
+      <td align="center">system.load1</td>
+    </tr>
+    <tr>
+      <td align="center">mem.slab</td>
+      <td align="center">system.load1_per_cpu</td>
+    </tr>
+    <tr>
+      <td align="center">mem.swap_cached</td>
```

Comment:
This one only has one row, it's missing a `<td>`. I think it would be better missing as the last row of the table instead of one at the middle